awx/api/views.py

```diff
@@ -1299,8 +1299,11 @@ def post(self, request, *args, **kwargs):
             if not project_update:
                 return Response({}, status=status.HTTP_400_BAD_REQUEST)
             else:
+                data = OrderedDict()
+                data.update(ProjectUpdateSerializer(project_update, context=self.get_serializer_context()).to_representation(project_update))
+                data['project_update'] = project_update.id
```

Comment:
oh, actually, it might be nice to put this line first, so the "project_update" key will be at the top of the response.

We have been paying little attention to ordering of the keys in the response, but it's something I want to look into